packages/ember-metal/lib/assert.js

```diff
@@ -0,0 +1,36 @@
+export let debugFunctions = {
+  assert() {},
+  warn() {},
+  debug() {},
+  deprecate() {},
+  deprecateFunc(_, fn) { return fn; },
```

Comment:
We just fixed the default (prod) version of this this morning, can you update to ensure the last arg is always returned?